include/machine.h

```diff
@@ -41,6 +50,7 @@ static inline p_region_t CONST pptr_to_paddr_reg(region_t reg)
 }
 
 #define paddr_to_pptr ptrFromPAddr
-#define pptr_to_paddr(x) addrFromPPtr(x)
+#define pptr_to_paddr addrFromPPtr
```

Comment:
Sould use explicit parameter names here.
```suggestion
#define pptr_to_paddr(x) addrFromPPtr(x)
```
Or maybe use one name only for everything to reduce confusion.

Reply:
The change here was to remove the reference to parameters and only update the token of the function name. Once again, I don't want to introduce an edit here that changes all the locations that these are called.